### ISSUE TITLE
Center site nav at widths < 900px

### ISSUE THREAD

**Issue by kevinwhoffman** [advancedwp/awp-2015#7]
Site nav is currently aligned left at smaller widths shown below. Page balance and composition could be improved by centering the nav at the 900px breakpoint.

![image](https://cloud.githubusercontent.com/assets/1938671/8769972/dfd96cb2-2e76-11e5-94f8-cf067e9390e4.png)

**Comment by advancedwpuser** [comment]
I'd actually like to implement the Menu with a slide-down on smaller width screens. Like this one:
http://codepen.io/surjithctly/pen/pLDwe

That would also mean centering the menu, and I like this method because it's Pure CSS, though a JS solution MIGHT have better browser support.

**Comment by kevinwhoffman** [comment]
Agreed a mobile nav solution is needed as well, but I was addressing the "tablet nav" scenario in between desktop and mobile - the point at which the logo pops above the nav and the nav slides below shown in the screenshot. IMO the full nav should be kept visible as long as possible before hiding it with a toggle.

Three breakpoints I'm proposing:
- palm - logo left-aligned, toggle right-aligned, nav revealed from above
- lap - logo top center-aligned, nav bottom center-aligned
- desk - logo left-aligned, nav right-aligned

**Comment by mathetos** [comment]
Since it's built on Foundation, and we're planning to implement Gulp/SASS we should do all breakpoints according to Foundation's breakpoint:
http://foundation.zurb.com/docs/media-queries.html

**Comment by kevinwhoffman** [comment]
Those were just conceptual breakpoints. Substitute palm/lap/desk with Foundation's small/medium/large breakpoints.
